### DISCUSSION TITLE
i-have-a-youtube-channel-now/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#221] [Announcements]
# i-have-a-youtube-channel-now/

This week I launched a YouTube channel. After many years of saying I would and not, I finally have. So here goes nothing! I'll be covering topics ranging such as self-hosting, home automation, Linux, cloud infrastructure automation tooling like Ansible and Terraform and much more... Here's my first video.

https://blog.ktz.me/i-have-a-youtube-channel-now/

**Comment by jamesbiederbeck** [comment]
You have a YouTube channel now? Then WTF was this? https://www.youtube.com/alexkretzschmar (PS, it's still the link on the youtube button on the top right of this page.)
